app/views/call_for_papers/_event_list.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="event-list" class="relative group border border-transparent md:hover:border-gray-200 rounded-xl flex flex-col gap-2 md:h-[75vh] md:overflow-scroll">
+  <% events.each do |event| %>
+    <%= link_to event.call_for_papers_link, class: "call-for-papers-item flex gap-4 group p-2 hover:bg-gray-200 rounded-md" do %>
+      <div class="flex flex-col lg:flex-row gap-8 items-center lg:justify-right text-center lg:text-left">
+        <%= image_tag image_path(event.avatar_image_path),
+              class: "rounded-xl border border-[#D9DFE3] size-16",
+              alt: "#{event.name} Avatar",
+              style: "view-transition-name: avatar",
+              loading: :lazy %>
+      </div>
+
+      <div class="flex-col flex justify-center relative overflow-hidden">
+        <div class="text-black group-hover:text-inherit font-bold text-xl" style="view-transition-name: title"><%= event.name %></div>
+        <div class="text-[#636B74] group-hover:text-inherit"><%= event.formatted_dates %> • <%= event.static_metadata.location %></div>
+        <div class="text-[#636B74] group-hover:text-inherit">CFP closes at <%= event.call_for_papers_deadline %></div>
```

Comment:
Nit: I think it would be great if we also match the date format we have elsewhere:

```suggestion
        <div class="text-[#636B74] group-hover:text-inherit">CFP closes at <%= event.call_for_papers_deadline.strftime("%B %d, %Y") %></div>
```